src/utils/index.js

```diff
@@ -661,39 +714,25 @@ export const getPriceEventForCrop = cropItem => ({
     getCropLifecycleDuration(cropItem) - PRICE_EVENT_STANDARD_DURATION_DECREASE,
 })
 
-/**
- * @param {Array.<Array.<?farmhand.plotContent>>} field
- * @param {function(?farmhand.plotContent)} condition
- * @returns {?farmhand.plotContent}
- */
 export const findInField = memoize(
-  (field, condition) => field.find(row => row.find(condition)) || null,
-  {
-    serializer: memoizationSerializer,
-  }
-)
-
-// This is currently unused, but it could be useful later.
-/**
- * @param {Array.<Array.<?farmhand.plotContent>>} field
- * @param {function(?farmhand.plotContent)} filterCondition
- * @returns {Array.<Array.<?farmhand.plotContent>>}
- */
-export const getCrops = memoize(
-  (field, filterCondition) =>
-    field.reduce((acc, row) => {
-      acc.push(...row.filter(filterCondition))
+  /**
+   * @param {(?farmhand.plotContent)[][]} field
+   * @param {function(?farmhand.plotContent): boolean} condition
+   * @returns {?farmhand.plotContent}
+   */
+  (field, condition) => {
+    const [plot = null] =
+      field.find(row => {
+        return row.find(condition)
+      }) ?? []
```

Comment:
This function was actually functionally broken before, as it didn't return what I intended. It simply hadn't manifested in a noticeable way yet. Typed JavaScript exposed a preexisting bug! 😃

Reply:
It appears that this change may have broken Scarecrows. A fix is at #431.